Demo/Demo/SwiftUIComponents/PayPalWeb/PayPalWebStatusView.swift

```diff
@@ -0,0 +1,64 @@
+import SwiftUI
+
+struct PayPalWebStatusView: View {
```

Comment:
We can also consider if we want different data displayed for each status - this PR matched what was previously displayed 1:1 but we may be able to reduce this down to a singular status view with the same data displayed across all portions of the flow.